code/modules/food_and_drinks/food/snacks/meat.dm

```diff
@@ -337,7 +337,7 @@
 
 /obj/item/reagent_containers/food/snacks/meat/steak/gondola
 	name = "gondola steak"
-	tastes = list("meat" = 1, "tranquility" = 1)
+	tastes = list("meat" = 1, /datum/reagent/tranquility = 1)
```

Comment:
These are just descriptions. Also reminds me some test for default named reagents would be nice.